asapdiscovery-data/asapdiscovery/data/scripts/download_fragalysis_data.py

```diff
@@ -17,7 +23,10 @@ def get_args():
 def main():
     args = get_args()
 
-    download(args.o, args.x)
+    # Overwrite the base target with the cli-specified target
+    API_CALL_BASE["target_name"] = args.target.capitalize()
```

Comment:
can't we make a copy rather than overwriting some base struct.

Reply:
Sounds good, I agree that would be a better approach. Will make the changes soon.

Reply:
Done in https://github.com/choderalab/asapdiscovery/pull/255/commits/e611ba001e35cea7a1577566737b63d335fd23ef